src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -781,6 +781,9 @@ public void CancelAllSubmissions()
 
         private void CancelAllSubmissions(bool async)
         {
+            ILoggingService loggingService = ((IBuildComponentHost)this).LoggingService;
+            loggingService.LogBuildCanceled();
```

Comment:
nit: The naming of the new event - it feels more as `CancelRequested` rather then `Canceled` based on usage here

Reply:
I'm following the existing pattern - Started, Finished, ... I would keep it that way for clarity.